docs/src/main/asciidoc/qute.adoc

```diff
@@ -243,9 +250,11 @@ import io.quarkus.qute.CheckedTemplate;
 @CheckedTemplate
 public class Templates {
     public static native TemplateInstance hello(String name); <1>
+    public static native TemplateInstance goodbye(String name); <2>
```

Comment:
Good catch, but this line needs also to be added in here:
https://github.com/quarkusio/quarkus/blob/f8e042f2caedeee44818d706980d4bc078e5c4e6/docs/src/main/asciidoc/qute.adoc?plain=1#L192-L195